t/ipcsysv.t

```diff
@@ -8,6 +8,10 @@
 #
 ################################################################################
 
+use strict;
+use warnings;
+
+our %Config;
```

Comment:
That shouldn't be necessary, right? Because %Config is imported\

Reply:
unfortunately this is required, `%Config` is used in the `BEGIN` block and parsed before

Steps to reproduce
```
echo 'use strict; use warnings; BEGIN { require Config; import Config; print $Config{d_sem} }' > test.pl
perl5.30.1 test.pl
Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at test.pl line 1.
BEGIN not safe after errors--compilation aborted at test.pl line 1
```

Reply:
for readability, you can also reproduce the issue with that simple script
```perl
use strict;
use warnings;

BEGIN {
  require Config; import Config;
  $Config{'d_sem'} and print q[ok\n];
}
```

Reply:
Or you just say `use Config;` before the begin block?

Reply:
sure but was trying to provide the minimal patch for adding strict/warnings and was avoiding refactor when unnecessary